### DISCUSSION TITLE
What is your workflow when you evaluate a new engine or DCC tool?

### DISCUSSION THREAD

**Discussion by AMZN-Liv** [o3de/o3de#2851] [Q&A]
Hey O3DE community!

I'm a product manager working on how we can improve the O3DE engine experience and am looking to learn more about how people make decisions around the tools that they use when evaluating a game engine or other digital content creation tool. What's the workflow that you use when you're testing out software to make games/3D applications for the first time? Do you try making a specific game type? Start with scripting? Get deep into look development?  

We're going to use this feedback to help us define our [Rev the Engine (RTE) workflow user stories (PDF link)](https://github.com/o3de/sig-ui-ux/blob/main/meetings/notes/July29MeetingNotes/Second-meeting-presentation.pdf), which is a framework that we're using to test different ways that people use the engine. You can learn more about Rev the Engine in the UI-UX SIG repo, or in the Discord channel!

**Comment by AMZN-Liv** [comment]
I'll share my answer as well to get things going. My background is in programming, so I spend more time on game mechanics than layout and look dev. As a result, I spend most of my time evaluating a new game engine by 1) making a new project, 2) creating a character controller to move around a space at runtime, and then going into 3) making a basic space game where I can fly around and script behaviors on the camera and do basic planet rotation and orbiting behaviors.

**Comment by HogJonny-AMZN** [comment]
I'm a Amazon Sr. Design Technologist.  Prior to o3de/lumberyard I worked on a lot of games, most were in-house engines, but we often evaluated to capabilities of other engines, and I have worked in both Unreal and Unity.

My experience as a Technical Art Director or even prior experience as a Art Lead/Director, the focus was on putting the pipelines and workflows through their paces with a outcome driven goal of presenting a beautiful corner (small interactive showcase).
- how well does the engine dovetail with the dcc tool ecosystem?
- how easy to learn are the in-engine tools, how low friction, how much pain?
- how long is a round trip iteration cycle: props, env, character, vehicle?
- how easily can the 'Look and feel' be customized, artistic vision acheived?
- how easily can we write new shaders, or implement new rendering techniques / visual styles?
- gap analysis, each artistic area: animation, vfx, post-fx, etc.

One good example is that I built a small interactive tech demo, prototypes it in our engine (shaders, etc.). Then ported the same content and experience to Unity to see how long that took, look for gaps, compare visuals, compare perf.  Then did the same with Unreal.
